### PR TITLE
Small note about how encryption is enabled and disabled

### PR DESCRIPTION
**Pull request by danielmarbach**
Connects to https://github.com/Particular/NServiceBus/pull/3991

### CODE REVIEW

File: nservicebus/security/encryption.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ The encryption algorithm used is [Rijndael](https://msdn.microsoft.com/en-us/lib
 
 Keep in mind that the security is only as strong as the keys; if the key is exposed, then an attacker can decipher the information. As such avoid encryption keys stored on the client (if deployed remotely) or even on a web server in the DMZ.
 
+NOTE: In NServiceBus Version 5 and below encryption is enabled by default and disables itself when there are no messages that require encryption. In NServiceBus Version 6 encryption is only enabled when explicitly configured with either `configuration.RijndaelEncryptionService` or `configuration.RegisterEncryptionService`.
```
Comment (SimonCropp) [inline nservicebus/security/encryption.md]: > NServiceBus Version 6 

should be 

> NServiceBus Versions 6 and above

Comment (SimonCropp) [inline nservicebus/security/encryption.md]: > NServiceBus Version 5 and below

should be 

> NServiceBus Versions 5 and below

Comment (SimonCropp) [inline nservicebus/security/encryption.md]: "configuration" is ambiguous. should be endpointConfiguration...

**Comment by danielmarbach** [comment]
@SimonCropp done

**Pull request merged**